docs/core/compatibility/sdk/10.0/prune-packagereference-privateassets.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: "Breaking change - PrunePackageReference marks direct prunable references with PrivateAssets=all and IncludeAssets=none"
+title: "Breaking change - PrunePackageReference privatizes direct prunable references"
 description: "Learn about the breaking change in the .NET 10 SDK where PrunePackageReference automatically marks directly prunable PackageReference with PrivateAssets=all and IncludeAssets=none."
-ms.date: 01/03/2025
+ms.date: 09/04/2025
 ai-usage: ai-assisted
 ---
 
-# PrunePackageReference marks direct prunable references with PrivateAssets=all and IncludeAssets=none
+# PrunePackageReference privatizes direct prunable references"
 
-The [PrunePackageReference](/nuget/consume-packages/package-references-in-project-files#prunepackagereference) feature automatically removes *transitive* packages that are provided by the target platform. With this change, the feature also marks *directly* prunable `PackageReference` items with `PrivateAssets=all` and `IncludeAssets=none` attributes. These attributes prevent the packages from appearing in generated dependency lists for packages.
+The [PrunePackageReference](/nuget/consume-packages/package-references-in-project-files#prunepackagereference) feature automatically removes *transitive* packages that are provided by the target platform. This pruning feature is enabled by default for projects that target or multi-target .NET 10. Now, the feature also marks *directly* prunable `PackageReference` items with `PrivateAssets=all` and `IncludeAssets=none` attributes. These attributes prevent the packages from appearing in generated dependency lists for packages.
 
 ## Version introduced
 
 .NET 10 Preview 7
 
 ## Previous behavior
 
-In earlier .NET 10 previews, directly prunable `PackageReference` items might have generated an [`NU1510` warning](/nuget/reference/errors-and-warnings/nu1510) but still appeared in the generated *.nuspec* dependencies for all target frameworks, even those where the package is provided by the platform.
+Prior to .NET 10, all `PackageReference` items appeared in the generated *.nuspec* dependencies for all target frameworks, even those where the package is provided by the platform. Starting in .NET 10 Preview 1, if pruning was enabled, directly prunable `PackageReference` items might have generated an [`NU1510` warning](/nuget/reference/errors-and-warnings/nu1510) but still appeared in the dependencies list.
```

Comment:
This needs some tweaks too. 
Privatization happens every time pruning is enabled, NU1510 will be raised when .NET 10 is one of the targets and the package can be removed from every framework it's specified in, effectively delete the reference from the project.